### ISSUE TITLE
pytest 6: pylint still shows not-callable error for builtin pytest marks even after #7473 and #7476

### ISSUE THREAD

**Issue by dgilland** [pytest-dev/pytest#7558]
I'm experiencing #7473 after the fix in #7476 with the following:

```python
import pytest

@pytest.mark.parametrize('a', [1, 2])
def test_a(a):
    pass
```

Where pylint still shows `not-callable` for `pytest.mark.parametrize`:

```
$ pylint -E test_pytest_7473.py
************* Module test_pytest_7473
test_pytest_7473.py:3:1: E1102: pytest.mark.parametrize is not callable (not-callable)
```

Environment:

- Python 3.8
- pytest==6.0.0
- pylint==2.5.3

FYI @bluetech

**Comment by The-Compiler** [comment]
Hmm, I can confirm: https://github.com/qutebrowser/qutebrowser/pull/5630/checks?check_run_id=920677338#step:8:18

**Comment by andy-maier** [comment]
I can confirm that too, using pytest 6.0.0 and pylint 2.4.4 (we pinned pylint to <2.5.0 for now due to some issues with it).
Travis run on Python 3.8: https://travis-ci.org/github/pywbem/pywbem/jobs/712820311

**Comment by bluetech** [comment]
I submitted a fix in #7565, will be backported to pytest 6.0.1 if merged.

Since I botched the previous attempt, would be good if any of the reports can confirm the fix. As a quick check you can just temporarily edit `venv/lib/python3.8/site-packages/_pytest/mark/structures.py` to match the PR, run pylint and see if it complains.

**Comment by dgilland** [comment]
I've confirmed that #7565 fixed the issue. Thanks @bluetech 👍
